scripts/ci_build_and_test_in_container.sh

```diff
@@ -34,13 +34,15 @@ Usage: $0
       Request for the build of geos only.
   --cmake-build-type ...
       One of Debug, Release, RelWithDebInfo and MinSizeRel. Forwarded to CMAKE_BUILD_TYPE.
+  --code-coverage
+      run a code build and test.
   --data-basename output.tar.gz
       If some data needs to be extracted from the build, the argument will define the tarball. Has to be a `tar.gz`.
   --exchange-dir /path/to/exchange
       Folder to share data with outside of the container.
   --host-config host-config/my_config.cmake
       The host-config. Path is relative to the root of the repository.
-  --install-dir-basename GEOS-e42ffc1
+  --install-dir-basename
```

Comment:
`GEOS-e42ffc1` was an example to emphasize the kind of information we were expecting.
Why did you remove it? At least, we would need a placeholder to indicate we're waiting for an argument.

Reply:
I thought I had accidentally pasted that in...haha.